Blocks/HTTP/test.cc

```diff
@@ -106,6 +107,74 @@ TEST(HTTPAPI, Register) {
   EXPECT_EQ(1u, HTTP(FLAGS_net_api_test_port).HandlersCount());
 }
 
+TEST(HTTPAPI, RegisterWithURLPathParams) {
+  using current::strings::Join;
+
+  HTTP(FLAGS_net_api_test_port).ResetAllHandlers();
+  auto handler = [](Request r) { r(r.url.path + ':' + Join(r.url_path_params, ',')); };
```

Comment:
Done in https://github.com/c5t/Current/commit/91cd01b19758c0a3e67b3f5be4d77caf10213bd6